### PR TITLE
feature/tests

### PR DESCRIPTION
**Pull request by Regiss05**
This PR should
- Configuration of jest test
- check tests test average of all the project

### CODE REVIEW

File: src/__Test__/Operate.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  expect(Operate(6, 9, '-')).toEqual('-3');
+});
+
+test('Test Operate Logic (Division) | Operate(9, 3, ÷)', () => {
+  expect(Operate(9, 3, '÷')).toEqual('3');
+});
+
+test('Test Operate Logic (Multiplication) | Operate(7, 3, x)', () => {
+  expect(Operate(7, 3, 'x')).toEqual('21');
+});
```
Comment (Temmarie) [inline src/__Test__/Operate.test.js]: - [x] Kindly ensure that all the functionalities & scenarios of the operate.js module are covered using Jest. You are currently missing a few functionalities, kindly add them and request a new review.
- [x] Also add more negative test cases, like `2/0`

File: src/__Test__/Calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+});
+
+test('Test Calculate Logic Calculate({ total: null, next: 4, operation: null}, ÷)', () => {
+  const state = { total: null, next: '4', operation: null };
+  expect(Calculate(state, '÷')).toEqual({ total: '4', next: null, operation: '÷' });
+});
+
+test('Test Calculate Logic Calculate({ total: 4, next: 982, operation: null}, AC)', () => {
+  const state = { total: '4', next: '982', operation: 'x' };
+  expect(Calculate(state, 'AC')).toEqual({ total: null, operation: null, next: null });
```
Comment (Temmarie) [inline src/__Test__/Calculate.test.js]: - [x] Good job adding a lot of tests, especially positive and negative case scenrios. Kindly ensure that all the functionalities & scenarios of the `calculate.js` module are covered using Jest.

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @Regiss05 and @misheck12    ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitFlow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 
- You've covered a considerable amount of functionalities   ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Recommended** ♻️
- [x] Kindly update your pull request message to be more descriptive. It should contain detailed information on the things done in this milestone. This helps you and other developers have an idea of the content of the pull request.

- [x] As requested in the project specifications:

- Run your full tests suite using Jest. All tests should pass. There's currently one test failing, `App.test.js`, kindly fix it.
- There should be the usage of Jest snapshots for testing all the components.
- All the functionalities & scenarios of the `operate.js` and `calculate.js` modules should be covered using Jest.
_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you have a limited number of reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

**Review by Kingstalux — APPROVED** [review]
Hi @team,
# Status: Approved ✔️ ✔️ ✔️  
128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

## Highlights 👏 
✅ All tests are passing 🔥 
✅ Good commit messages 👍 

Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
 
Congratulations! 🎉 

- [ ] (OPTIONAL) Kindly update the author's section of your README file to include all the authors that worked on this project 👍 This will make your README file more professional 💯 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
